stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/ImageSlidePageFragment.kt

```diff
@@ -0,0 +1,36 @@
+package io.getstream.chat.android.ui.images
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.getstream.sdk.chat.ImageLoader
+import io.getstream.chat.android.ui.databinding.StreamItemImageGalleryBinding
+
+public class ImageSlidePageFragment : Fragment() {
+
+    public var imageUrl: String? = null
+
+    private lateinit var binding: StreamItemImageGalleryBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return StreamItemImageGalleryBinding.inflate(LayoutInflater.from(context)).apply {
+            binding = this
+        }.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        imageUrl?.let {
+            ImageLoader.run {
```

Comment:
IMO, import is better
```
imageUrl?.let {
    binding.ivImageItem.load(it)
}
```

Reply:
I think it's fine to pass null here `binding.ivImageItem.load(nullable)`

Reply:
No problem to pass a nullable. But I need the `run` to change the scope to I can use the extension functions of ImageLoader.

Reply:
You can import it with `import com.getstream.sdk.chat.ImageLoader.load` to make it work cleanly.

Though it was probably a bad design choice to make it a member extension in the first place, member extensions are weird. We could move these into top level extensions instead, if we want to.

Reply:
done